specification/applicationinsights/data-plane/Monitor.Exporters/preview/2020-09-15/swagger.json

```diff
@@ -0,0 +1,1044 @@
+              },
+              "baseData": {
+                  "$ref": "#/definitions/Domain",
+                  "description": "The data payload for the telemetry request"
+              }
+          }
+      },
+      "DataPoint" : {
+        "x-ms-client-name": "MetricDataPoint",
+          "type": "object",
+          "description": "Metric data single measurement.",
+          "required": [
+              "name",
+              "value"
+          ],
+          "properties": {
+              "ns": {
+                  "x-ms-client-name": "namespace",
+                  "type": "string",
+                  "description": "Namespace of the metric.",
+                  "maxLength": 256
+              },
+              "name": {
+                  "type": "string",
+                  "description": "Name of the metric.",
+                  "maxLength": 1024
+              },
+              "kind": {
+                  "x-ms-client-name": "dataPointType",
+                  "$ref": "#/definitions/DataPointType",
+                  "description": "Metric type. Single measurement or the aggregated value."
+              },
+              "value": {
+                  "type": "number",
+                  "description": "Single value for measurement. Sum of individual measurements for the aggregation.",
+                  "format": "double"
+              },
+              "count": {
+                  "type":"number",
+                  "format": "int32",
+                  "description": "Metric weight of the aggregated metric. Should not be set for a measurement.",
+                  "x-nullable": true
+              },
+              "min": {
+                  "type":"number",
+                  "format": "double",
+                  "description": "Minimum value of the aggregated metric. Should not be set for a measurement.",
+                  "x-nullable": true
+              },
+              "max": {
+                  "type":"number",
+                  "format": "double",
+                  "description": "Maximum value of the aggregated metric. Should not be set for a measurement.",
+                  "x-nullable": true
+              },
+              "stdDev": {
+                  "type":"number",
+                  "format": "double",
+                  "description": "Standard deviation of the aggregated metric. Should not be set for a measurement.",
+                  "x-nullable": true
+              }
+          }
+      },
+      "DataPointType" : {
+          "type": "string",
+          "description": "Type of the metric data measurement.",
+          "enum": [
+              "Measurement",
+              "Aggregation"
+          ]
+      },
+      "Domain" : {
+          "x-ms-client-name": "MonitorDomain",
+          "type": "object",
+          "description": "The abstract common base of all domains.",
+          "properties": {
+              "test": {
```

Comment:
what's the point of this property?

Reply:
After trying variations of this swagger file with the JS exporter, I think this `test` field is blocking JS serialization. I was able to get something running by removing the entire `properties` field for `Domain`. Is `test` required for this to work in Java/C#?

Reply:
Removed `test` from Domain and moved `ver` which is common for all sub-classes of domain here.

Reply:
Even with `ver` here, serialized JS `baseData` only contain a `ver` field, and everything else is dropped. I've tried adding `"additionalProperties": true` as well. This seems to be a supported scenario, and so is this just a JS (and maybe Python) generation issue?
![image](https://user-images.githubusercontent.com/5515583/93804855-bb8a7d80-fbfb-11ea-9c8b-3cee6211545a.png)